example_auto_set_url.cmake

```diff
@@ -0,0 +1,5 @@
+set(PICO_EXTRAS_URL_BASE "https://github.com/raspberrypi/pico-extras/tree/HEAD")
```

Comment:
I suspect you want `pico-playground` here (i.e. the repo you're adding this PR to), rather than `pico-extras`?

Reply:
Yes!! I got confused here. I just fixed this is a new commit! :)

Reply:
You probably want to also rename `PICO_EXTRAS_URL_BASE` to `PICO_PLAYGROUND_URL_BASE` :wink: